godtools/App/Features/Tools/SwiftUI/AllTools/ToolCategoriesView/ToolCategoryButtonView/ViewModels/ToolCategoryButtonViewModel.swift

```diff
@@ -12,29 +12,38 @@ class ToolCategoryButtonViewModel: BaseToolCategoryButtonViewModel {
     
     // MARK: - Properties
     
-    let attrCategory: String
-    let localizationServices: LocalizationServices
-    let languageSettingsService: LanguageSettingsService
+    let category: ToolCategoryDomainModel
+    private let localizationServices: LocalizationServices
+    private let getSettingsPrimaryLanguageUseCase: GetSettingsPrimaryLanguageUseCase
         
     // MARK: - Init
     
-    init(attrCategory: String, selectedAttrCategory: String?, localizationServices: LocalizationServices, languageSettingsService: LanguageSettingsService) {
-        self.attrCategory = attrCategory
+    init(category: ToolCategoryDomainModel, selectedCategoryName: String?, localizationServices: LocalizationServices, getSettingsPrimaryLanguageUseCase: GetSettingsPrimaryLanguageUseCase) {
+        self.category = category
         self.localizationServices = localizationServices
-        self.languageSettingsService = languageSettingsService
+        self.getSettingsPrimaryLanguageUseCase = getSettingsPrimaryLanguageUseCase
         
-        let bundle = localizationServices.bundleLoader.bundleForPrimaryLanguageOrFallback(in: languageSettingsService)
-        let translatedCategory = localizationServices.toolCategoryStringForBundle(bundle: bundle, attrCategory: attrCategory)
+        let bundle: Bundle
+        if let primaryLanguage = getSettingsPrimaryLanguageUseCase.getPrimaryLanguage() {
```

Comment:
Hey @rachaelblue I would like this logic to go in the ```GetToolCategoriesUseCase```.  The ```ToolCategoryDomainModel``` title would then map to the button title.